tock/tock/settings/base.py

```diff
@@ -23,6 +23,8 @@
 LOGIN_URL = '/auth/login'
 LOGIN_REDIRECT_URL = '/'
 
+CSRF_FAILURE_VIEW = 'tock.views.csrf_failure'
```

Comment:
Whoa, I had no idea [`CSRF_FAILURE_VIEW`](https://docs.djangoproject.com/en/2.0/ref/settings/#csrf-failure-view) was even a setting! Thanks for teaching me something here, and for making everything more <img src="https://i2.wp.com/sharkingforchipsanddrinks.com/wp-content/uploads/2016/01/Django-Bango-Final-Logo-sm.png?resize=500%2C500" alt="Django Bango">

Reply:
+1